stock_picking_mail_incorrect_qty/data/mail_template.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Mail template are declared in a NOUPDATE block
+         so users can freely customize/delete them -->
+    <data noupdate="0">
+        <record id="mail_template_incorrect_delivery" model="mail.template">
+            <field name="name">Incorrect Delivery</field>
+            <field name="email_from">
+                ${ctx.get('company').email|safe}
+            </field>
+            <field name="subject">Incorrect Delivery (${object.name})</field>
+            <field name="partner_to">${ctx.get('company').email|safe}</field>
```

Comment:
On bees database, it will send the email at `membre@bees-coop.be`. It's probably a good default value but it may be a good idea to say in `usage.rst` to configure this template with the good recipient.